### PR TITLE
Adds Brazilian regional event: Caipyra 2016

### PR DESCRIPTION
**Pull request by jtemporal**
The first edition of [Caipyra](http://2016.caipyra.python.org.br/) happened in 2016 and here are the videos

### CODE REVIEW

File: caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Anota\u00e7\u00f5es da palestra: http://piratepad.net/caipyra-armando\nhttp://caipyra.python.org.br",
```
Comment (Daniel-at-github) [inline caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json]: http://piratepad.net/caipyra-armando seems empty

**Comment by Daniel-at-github** [comment]
Making some minor changes, will merge it upstream.

File: caipyra-2016/videos/de-professor-do-ensino-fundamental-a-desenvolvedor-python-matheus-godoy.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Palestra do Matheus Godoy no Caipyra 2016:\n\nDe professor do ensino fundamental a desenvolvedor python\n\nSlides: bit.ly/caipyra2016",
```
[the remark this answers is not in the document]
Comment (jtemporal) [reply caipyra-2016/videos/de-professor-do-ensino-fundamental-a-desenvolvedor-python-matheus-godoy.json]: yep! here you go: https://docs.google.com/presentation/d/e/2PACX-1vTPNCVMAHM4uMJLfm1qQSTD19k7_gVMZMw-LxQmn_UNo8ihVWxs0n3Q0vKNpY4rNiXdyB0TIXVgGWVB/pub?start=false&loop=false&delayms=3000

File: caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json (hunk 1 above)
Comment (jtemporal) [reply caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json]: Yep! Armando did not use slides that day, and the piratepad went blank shortly after it

**Comment by Daniel-at-github** [comment]
Thanks for submitting Caipyra 2016!
Feel free to submit/improve data in pyvideo.

As data scientist, if you see a way to improve the ETL or the data cleaning of the site, it will be more than wellcome.
Lowering the effort and the entry barrier to contribute to pyvideo is what makes it sustainable.

File: caipyra-2016/videos/de-professor-do-ensino-fundamental-a-desenvolvedor-python-matheus-godoy.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Daniel-at-github) [reply caipyra-2016/videos/de-professor-do-ensino-fundamental-a-desenvolvedor-python-matheus-godoy.json]: Done

File: caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json (hunk 1 above)
Comment (Daniel-at-github) [reply caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json]: Done

**Comment by jtemporal** [comment]
This is awesome! Over the weekend I'll be working with a group of women to submit more Brazilian events that happened over the past years. I'll take notes on how to improve the ETL and cleaning data process if see any more room for improvement. Thanks!

**Pull request merged**